src/commcare_cloud/ansible/roles/pgbouncer/templates/pgbouncer2.init.j2

```diff
@@ -0,0 +1,100 @@
+#!/bin/sh
```

Comment:
how is this file different from `pgbouncer.init.j2`? If it's not can they just use the same template?

Reply:
in this one the name and binary are different (it still calls `/usr/sbin/pgbouncer` but needs to use pgbouncer2 as the name for the config file and other variables). It is possible these could be combined to be shared if the binary variable `BIN` were hardcoded and the the `NAME` were passed in as a variable